native/src/routes/sprungbrett/containers/SprungbrettOfferContainer.js

```diff
@@ -110,29 +112,29 @@ class SprungbrettOfferContainer extends React.Component<SprungbrettPropsType, Sp
     const { jobs, error, timeoutExpired } = this.state
 
     if (error) {
-      return <ScrollView refreshControl={<RefreshControl onRefresh={this.loadSprungbrett} refreshing={false} />}
-                         contentContainerStyle={{ flexGrow: 1 }}>
-        <FailureContainer errorMessage={error.message} tryAgain={this.loadSprungbrett} />
-      </ScrollView>
+      return (
+        <LayoutedScrollView refreshControl={<RefreshControl onRefresh={this.loadSprungbrett} refreshing={false} />}>
```

Comment:
LayoutContainer was added here implicitly. Is that correct?

Reply:
Yes, see answer above.